u8timeseries/timeseries.py

```diff
@@ -425,6 +441,84 @@ def append(self, other: 'TimeSeries') -> 'TimeSeries':
             conf_hi = self._confidence_hi.append(other.conf_hi_pd_series())
         return TimeSeries(series, conf_lo, conf_hi)
 
+    def add(self, values: np.ndarray, index: pd.DatetimeIndex = None,
```

Comment:
the name `add` can be a bit confusing. How about something different, like `append_values`?

Reply:
It seems more understandable that way. I will change it